src/pint/scripts/photonphase.py

```diff
@@ -212,9 +206,8 @@ def main(argv=None):
 
     # Compute model phase for each TOA
     iphss, phss = modelin.phase(ts, abs_phase=True)
-    # ensure all postive
-    negmask = phss < 0.0
-    phases = np.where(negmask, phss + 1.0, phss)
+    phases = phss.value
+    phases[phases < 0.0] += 1
```

Comment:
is `phases` guaranteed to be between -1 and 1? `phases %= 1` should accomplish the same thing whether this is so or not.

Reply:
Yes, the PINT convention is that the fractional part is on [-0.5, 0.5), or possibly (-0.5,0.5].  I suppose mod is safer, but I hate to use it when a simple add will do!

Reply:
If it soothes your discomfort, floating-point operations are generally faster than branches and out-of-order memory access...

Reply:
I just did some crude timing with a big array and they are more similar than I would have thought:
```
In [22]: a = u.Quantity(np.random.standard_normal(1000000))                                                   

In [23]: %timeit "phases = a.value ; phases[phases < 0.0] += 1"                                               
8.14 ns ± 1.13 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [24]: a = u.Quantity(np.random.standard_normal(1000000))                                                   

In [25]: %timeit "a %= 1"                                                                                     
9.38 ns ± 0.801 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)
```

Reply:
Alright, mod it is!

Reply:
Yeah, I think an in-place mod is probably a bit nicer.  There are probably a few other places in PINT where that can be changed....